src/components/SideNav/MinimizedCategories.svelte

```diff
@@ -46,4 +53,10 @@
     fill: var(--waterloo);
     margin: 0 0 16px;
   }
+
+  .btn.active {
+    --bg: var(--white);
+
+    border-radius: 6px;
+  }
```

Comment:
`[D-0-1]` Not needed in the master branch. You can merge the master to get the updates